data/gui/screens/help1.stkgui

```diff
@@ -54,28 +54,28 @@
                     <spacer width="25" height="25"/>
                     <bubble proportion="3" height="100%" word_wrap="true"
                             I18N="In the help menu"
-                            text="Collecting nitro allows you to get speed boosts whenever you wish by pressing the appropriate key or button. You can see your current level of nitro in the gauge at the bottom-right of the race screen."/>
-                    </div>
-
-                <div width="100%" proportion="4" layout="horizontal-row">
-                    <icon align="center" width="64" height="64" icon="gui/icons/gui_lock.png"/>
+                            text="Collecting nitro allows you to use small speed boosts by pressing the assigned key or button*. You can see your current level of nitro in the gauge at the bottom-right of the race screen."/>
```

Comment:
You can use nitro to get a speed boost, you can't use a speed boost.

I don't see the point of writing "small". In SuperTux while skidding, nitro still gives +17% of speed. In Novice, it gives +33% of speed. The duration depends of how much nitro you have, but isn't that bad. In my book, that doesn't deserve to be called small.